portal-frontend/src/app/features/notice-of-intents/edit-submission/parcels/parcel-entry/parcel-entry.component.html

```diff
@@ -0,0 +1,374 @@
+          [disabled]="_disabled"
+        ></app-parcel-owners>
+      </div>
+    </ng-container>
+    <ng-container *ngIf="isCrownLand">
+      <div>
+        <label for="crown-owner-type">Select owner:</label>
+      </div>
+      <mat-radio-group
+        formControlName="crownLandOwnerType"
+        class="crown-owner-type"
+        id="crown-owner-type"
+        aria-label="Select an Owner"
+      >
+        <mat-radio-button value="provincial">Provincial Crown</mat-radio-button>
+        <mat-radio-button value="federal">Federal Crown</mat-radio-button>
+        <div *ngIf="showErrors && !crownLandOwnerType.getRawValue()" class="field-error">
+          <mat-icon>warning</mat-icon>
+          <div>This field is required</div>
+        </div>
+      </mat-radio-group>
+      <mat-form-field class="full-width-input" appearance="outline">
+        <input
+          id="government-search"
+          type="text"
+          aria-label="Text"
+          matInput
+          placeholder="Search by Ministry or Department"
+          [matAutocomplete]="auto"
+          (input)="onTypeOwner($event)"
+          [formControl]="ownerInput"
+        />
+        <mat-autocomplete #auto="matAutocomplete">
+          <mat-option
+            (click)="onSelectOwner(option, option.isSelected)"
+            *ngFor="let option of filteredOwners"
+            [disabled]="option.isSelected"
+          >
+            <div class="owner-option">
+              <div>{{ option.displayName }}</div>
+              <button mat-flat-button color="primary" [disabled]="option.isSelected">
+                {{ option.isSelected ? 'Added' : 'Add' }}
+              </button>
+            </div>
+          </mat-option>
+          <mat-option
+            class="option"
+            *ngIf="filteredOwners.length === 0 && enableAddNewOwner"
+            (click)="onAddNewGovernmentContact()"
+          >
+            <div class="owner-option">
+              <div>No owner matching search</div>
+              <button mat-flat-button color="primary">Add new government contact</button>
+            </div>
+          </mat-option>
+          <mat-option class="option" *ngIf="owners.length > 0" (click)="onSeeAllOwners()">
+            <span class="link-text">See all Owners</span>
+          </mat-option>
+        </mat-autocomplete>
+      </mat-form-field>
+      <div *ngIf="ownerInput.errors && ownerInput.errors['required']" class="field-error">
+        <mat-icon>warning</mat-icon>
+        <div *ngIf="ownerInput.errors?.['required']">This field is required</div>
+      </div>
+      <button
+        class="new-owner"
+        *ngIf="enableAddNewOwner"
+        (click)="onAddNewGovernmentContact()"
+        mat-stroked-button
+        color="primary"
+        [disabled]="_disabled || parcel.owners.length > 0"
+      >
+        Add new government contact
+      </button>
+      <div>
+        <!--        <app-parcel-owners-->
```

Comment:
I thought you were going to enable this once owners are ready

Reply:
Good catch!